.github/workflows/build-test.yml

```diff
@@ -22,9 +22,6 @@ jobs:
         with:
           fetch-depth: 0
       - uses: ./.github/actions/pnpm-setup-action
-        # Portals won't build without a configuration. For now, copy the test configuration.
-        # We may want to invert the model and give each portal its own project in the workspace, so we can build the portals in parallel.
-      - run: pnpm nx run portals:copy-test-configuration
```

Comment:
This is no longer necessary, since each configuration is now in its own workspace.